client/verta/.pylintrc

```diff
@@ -1,331 +1,323 @@
-[MASTER]
```

Comment:
I wouldn't recommend reviewing this whole file, or even necessarily giving this PR a close review at all—it's autogenerated from `pylint --generate-rcfile`, and lots of items have simply been moved around.

I've commented on project-specific restorations I've made.